src/main/java/seedu/taassist/commons/core/Messages.java

```diff
@@ -9,5 +9,6 @@ public class Messages {
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_STUDENT_DISPLAYED_INDEX = "The student index provided is invalid";
     public static final String MESSAGE_STUDENTS_LISTED_OVERVIEW = "%1$d students listed!";
-    public static final String MESSAGE_MODULE_CLASS_DOES_NOT_EXIST = "The module class does not exist";
+    public static final String MESSAGE_CLASS_DOES_NOT_EXIST = "The class does not exist: %1$s";
+    public static final String MESSAGE_SOME_CLASSES_DO_NOT_EXIST = "One or more classes do not exist.";
```

Comment:
After seeing @njxue's PR #102, I prefer that solution more. 
```
public static final String MESSAGE_MODULE_CLASS_DOES_NOT_EXIST = "A provided module class does not exist\n"
            + "Existing classes: %1$s";
```
I will update this after #102 is merged.